src/Datadog.Trace.ClrProfiler.Native/cor_profiler.cpp

```diff
@@ -351,6 +364,32 @@ HRESULT STDMETHODCALLTYPE CorProfiler::ModuleLoadFinished(ModuleID module_id,
        module_info.assembly.name == "System.Private.CoreLib"_W)) {
     corlib_module_loaded = true;
     corlib_app_domain_id = app_domain_id;
+    
+    ComPtr<IUnknown> metadata_interfaces;
+    auto hr = this->info_->GetModuleMetaData(module_id, ofRead | ofWrite, IID_IMetaDataImport2, metadata_interfaces.GetAddressOf());
+    
+    // Get the IMetaDataAssemblyImport interface to get metadata from the
+    // managed assembly
+    const auto assembly_import = metadata_interfaces.As<IMetaDataAssemblyImport>(IID_IMetaDataAssemblyImport);
+    const auto assembly_metadata = GetAssemblyImportMetadata(assembly_import);
+
+    hr = assembly_import->GetAssemblyProps(
+        assembly_metadata.assembly_token, &corAssemblyProperty.ppbPublicKey,
+        &corAssemblyProperty.pcbPublicKey, &corAssemblyProperty.pulHashAlgId,
+        NULL, 0, NULL, &corAssemblyProperty.pMetaData,
+        &corAssemblyProperty.assemblyFlags);
+
+    if (FAILED(hr)) {
+      Warn("AssemblyLoadFinished failed to get properties for COR assembly ");
+    }
+
+    corAssemblyProperty.szName = module_info.assembly.name;
+
+    Info("COR library: ", corAssemblyProperty.szName, " ",
```

Comment:
If you want to keep this log line around, we can change it to a debug log

Reply:
Don't you think is a good info to have in the log as default? I think the COR lib version could give us a hint of the .NET version the client is using.

Reply:
Yeah that's a good point, and this will only get called once per application. Let's keep it 👍